src/Microsoft.DotNet.XHarness.iOS.Shared/Listeners/SimpleListenerFactory.cs

```diff
@@ -61,6 +64,9 @@ public class SimpleListenerFactory : ISimpleListenerFactory
 
         switch (transport)
         {
+            case ListenerTransport.Relay:
```

Comment:
I don't follow - is this value actually set above? Or this branch will never trigger?